src/ui/Pricing/SubscriptionSettings/SubscriptionCard/Card.svelte

```diff
@@ -147,15 +145,62 @@
     display: inline-flex;
     --bg: var(--primary, var(--green));
     --bg-hover: var(--primary-hover, var(--green-hover));
+    --v-padding: 8px;
   }
 
   .btn-2 {
     --bg: var(--white);
+    --v-padding: 7px;
+  }
+
+  .subaction {
+    margin-left: 12px;
   }
 
   .billing {
     position: absolute;
     bottom: 16px;
     right: 16px;
   }
+
+  .label {
+    font-size: 12px;
+  }
```

Comment:
This breaks desktop version design. Correct typography requires font-size and line-height to be applied accordingly.